packages/features/ee/dsync/lib/handleUserEvents.ts

```diff
@@ -69,11 +69,6 @@ const handleUserEvents = async (event: DirectorySyncEvent, organizationId: numbe
     select: dSyncUserSelect,
   });
 
-  // User is already a part of that org
-  if (user && UserRepository.isAMemberOfOrganization({ user, organizationId }) && eventData.active) {
```

Comment:
It was causing the update to not sync attribues as the user is already a part of org.